.eslintrc

```diff
@@ -43,13 +43,13 @@
         "omitLastInOneLineBlock": true
       }
     ],
-    "no-trailing-spaces": 0,
+    "no-trailing-spaces": 1,
     "no-else-return": 2,
     "no-extra-bind": 0,
     "no-implicit-coercion": 0,
     "no-useless-call": 0,
     "no-return-assign": 0,
-    "eol-last": 0,
+    "eol-last": 1,
```

Comment:
It this change necessary? This will cause a new line to be added for every file in subsequent commits.

Reply:
Yes, and I think that is the styling we follow, it was just not added as an eslint rule, so some files have the new line at the end and some don't.

Reply:
Yes you are right, okay looks good to me